melodies_monet/plots/aircraftplots.py

```diff
@@ -0,0 +1,1241 @@
+
+# Copyright (C) 2022 National Center for Atmospheric Research and National Oceanic and Atmospheric Administration
+# SPDX-License-Identifier: Apache-2.0
+#
+#Code to create plots for surface observations
```

Comment:
I seem some functions in here that I believe are (also) defined elsewhere in MM. Could they be imported instead?

Reply:
@zmoon Can you provide any suggested template/example I should follow to achieve that?

Reply:
For example, if you want `new_color_map` from `surfplots` you can use
```python
from .surfplots import new_color_map
```